docs/Inputs.md

```diff
@@ -522,6 +522,26 @@ import { FormDataConsumer } from 'react-admin';
  );
 ```
 
+**Important**: By default `react-hook-form` submits values of unmounted input components, so if you don't want this behaivour you can set `shouldUnregister` prop to `true` in the form
```

Comment:
I'd put the shouldUnregister directly in the first example (L513) instead of duplicating it. then, you can explain why it's necessary.